.github/workflows/delivery/docker/buildpack.yml

```diff
@@ -0,0 +1,16 @@
+go:
+  targets:
+    - ./cmd/pack
+
+  build:
+    # The go.build.flags property allows you to override the default build
+    # flags when compiling your program.
+    flags:
+      - -buildmode=default
+      - -tags=paketo
+      - -ldflags="-s -w -X 'github.com/buildpacks/pack.Version=#{PACK_VERSION}#' -extldflags ${LDFLAGS}"
+
+    # The go.build.import-path property allows you to specify an import path
+    # for your application. This is necessary if you are building a $GOPATH
+    # application that imports its own sub-packages.
+    import-path: github.com/buildpacks/pack
```

Comment:
You should not need this as this is a `go.mod` app. The buildpack can find the import path from the `go.mod` file.